taskcluster/utils.py

```diff
@@ -268,6 +270,45 @@ def isExpired(certificate):
   return expiry < int(time.time() * 1000 + 20 * 60 * 1000)
 
 
+def readCfgFile():
```

Comment:
How about allowing config file as parameter, and defaulting it to `CONFIG_FILE`...

Maybe rename this `readCredentialsFile()`... I think we can make it a convention across client libs to use JSON format in `taskcluster.json` file, under `user_data_dir`...